app/models/customer.rb

```diff
@@ -0,0 +1,15 @@
+class Customer < ApplicationRecord
+    belongs_to :supermarket
+    has_many :customer_items
+    has_many :items, through: :customer_items
+
+    # do I use a .self here?
+    def add_items(item_id)
+        @customer = Customer.find(params[:id])
```

Comment:
This is the content that could go in a `create` action in your customer_items_controller